package.json

```diff
@@ -16,6 +16,7 @@
     "clean:install": "pnpm clean:node_modules && pnpm install --frozen-lockfile",
     "build": "pnpm clean:bundle && turbo ready && turbo build",
     "build:firefox": "pnpm clean:bundle && turbo ready && cross-env __FIREFOX__=true turbo build",
+    "build:eslint": "esbuild eslint.config.ts --outfile=eslint.config.js",
```

Comment:
It this case, when you have `tsconfig` you don't really need `esbuild` you can build it via `tsc`
I will probably remove all `esbuild` from entire project to simplify it, because it's harder and harder to maintain.

Let me know

Reply:
Implemented this in the latest change